### DISCUSSION TITLE
App doesn't allow me to set details on issues

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [community/community#38307] [Mobile] [answered]
I'm helping on the PowerToys repo with triaging and such. App still doesn't allow me to set labels or milestones and so on.
Using Android 11 on Fairphone 3, v. 1.86.0
PLEASE. FIX. THIS.

> **Note**
> This does not apply to issues in a repo that is _mine_

![Screenshot_20221104-204124.png](https://user-images.githubusercontent.com/65828559/200064591-99a509d4-c5ab-4f79-a49e-b8eca6e932e2.png)

**Comment by aconsuegra** [comment, marked as answer]
Hi @Jay-o-Way, thanks for getting in touch, and sorry about the hassle. We recently shipped some improvements to the way setting labels works in the Android app, so I would appreciate if you you could give that a try and see if it works for you. The changes are included in our latest beta (1.89.0).

**Reply by Jay-o-Way** [reply]
Tanks for your message @aconsuegra. I'm using 1.86 now and I'd be happy to try a beta version, but **you gotta tell me where to get it**, because it is impossible to find. There's only one single app from GitHub as developer in G. Play Store: https://play.google.com/store/apps/developer?id=GitHub&hl=en_US&gl=US

**Reply by aconsuegra** [reply]
If you visit the app screen in Google Play from your Android device, you should see a button to join the Beta testing program. Let me know if that works for you.

**Reply by Jay-o-Way** [reply]
Okay, I had to search for a bit and then installed the update. Can confirm that the beta version properly deals with the labels. Thanks.
So, that leaves one question: Any idea how long 'til v 1.69 is publicly available? 😇

**Reply by aconsuegra** [reply]
Thanks for confirming! The issue is probably still there for stuff like milestones, but we will work on fixing that one too. 1.89.0 should be publicly available next week.
